src/components/Page/Page.tsx

```diff
@@ -94,7 +95,20 @@ export class Page extends React.PureComponent<ComposedProps, never> {
     );
   }
 
-  private hasHeaderContent() {
+  private get renderWithAppBridge(): boolean {
+    const {
+      polaris: {appBridge},
```

Comment:
`polaris` looks unused?

Reply:
Here we're just pulling out `appBridge` from `polaris.`